packages/checkout/widgets-lib/src/lib/walletConnect.ts

```diff
@@ -149,4 +159,31 @@ export class WalletConnectManager {
       }
     });
   }
+
+  private async loadWalletListings(): Promise<Response | undefined> {
+    // eslint-disable-next-line max-len
+    const walletListingsApi = `https://explorer-api.walletconnect.com/v3/wallets?projectId=${this.walletConnectConfig.projectId}&ids=${metamaskId},${frontierId},${coinbaseId}`;
+    try {
+      const response = await fetch(walletListingsApi);
+      const data = await response.json();
+      return data;
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('Error fetching wallet listings', error);
+    }
+    return undefined;
+  }
```

Comment:
This is great 👏 . It might be worth saving/caching this response so we don't have to call it everytime we want call getWalletLogoUrl.